src/Configuration/Config.FileExtensions/src/FileConfigurationProvider.cs

```diff
@@ -125,7 +126,7 @@ private void HandleException(Exception e)
             }
             if (!ignoreException)
             {
-                throw e;
+                ExceptionDispatchInfo.Capture(e).Throw();
```

Comment:
Shouldn't the capture happen on line 90?
https://github.com/aspnet/Extensions/pull/1907/files#diff-61c1423e04275a1e7c00297557873270R90

Reply:
Yeah, updated to pass in the dispatch info instead and updated the other place this gets called